### ISSUE TITLE
ImportError: cannot import name 'GuideSheet'

### ISSUE THREAD

**Issue by reagle** [saulpw/visidata#2560]
Hi, I thought I'd try [uv](https://github.com/astral-sh/uv) this weekend and installed visidata but I end up getting an error.

```
❯ uv tool install visidata --with lxml --with zstandard
Resolved 7 packages in 169ms
Prepared 1 package in 532ms
Installed 2 packages in 3ms
 + lxml==5.3.0
 + zstandard==0.23.0
Installed 3 executables: vd, vd2to3.vdx, visidata
 ~
❯ vd
Traceback (most recent call last):
  File "/Users/reagle/.local/bin/vd", line 6, in <module>
    visidata.main.vd_cli()
    ^^^^^^^^^^^^^^^^^^^^
```

Has anyone with more clue/experience tried visidata with uv?

**Comment by reagle** [comment]
Also, one should be able to simply run it with uvx.

```
❯ uvx visidata
Installed 5 packages in 8ms
Traceback (most recent call last):
  File "/Users/reagle/.cache/uv/archive-v0/M2Rslfs0mZFGrNRElBxLC/bin/visidata", line 7, in <module>
    from visidata.main import vd_cli
  File "/Users/reagle/.cache/uv/archive-v0/M2Rslfs0mZFGrNRElBxLC/lib/python3.12/site-packages/visidata/__init__.py", line 157, in <module>
    importFeatures()
  File "/Users/reagle/.cache/uv/archive-v0/M2Rslfs0mZFGrNRElBxLC/lib/python3.12/site-packages/visidata/__init__.py", line 139, in importFeatures
    vd.importSubmodules('visidata.features')
  File "/Users/reagle/.cache/uv/archive-v0/M2Rslfs0mZFGrNRElBxLC/lib/python3.12/site-packages/visidata/settings.py", line 508, in importSubmodules
    vd.importModule(pkgname + '.' + module.name)
  File "/Users/reagle/.cache/uv/archive-v0/M2Rslfs0mZFGrNRElBxLC/lib/python3.12/site-packages/visidata/settings.py", line 491, in importModule
    r = importlib.import_module(pkgname)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/reagle/.cache/uv/archive-v0/M2Rslfs0mZFGrNRElBxLC/lib/python3.12/site-packages/visidata/features/errors_guide.py", line 1, in <module>
    from visidata import GuideSheet, vd
ImportError: cannot import name 'GuideSheet' from partially initialized module 'visidata' (most likely due to a circular import) (/Users/reagle/.cache/uv/archive-v0/M2Rslfs0mZFGrNRElBxLC/lib/python3.12/site-packages/visidata/__init__.py)
```

**Comment by reagle** [comment]
Opened an issue at uv as well. https://github.com/astral-sh/uv/issues/8184

**Comment by reagle** [comment]
It looks like my experiment with uv might've coincided with 3.1 release: `pip intall visdata` has the same problem.

```
❯ pip install visidata
Collecting visidata
  Using cached visidata-3.1-py3-none-any.whl.metadata (5.3 kB)
Requirement already satisfied: python-dateutil in ./.pyenv/versions/3.12.7/lib/python3.12/site-packages (from visidata) (2.9.0.post0)
Requirement already satisfied: importlib-metadata>=3.6 in ./.pyenv/versions/3.12.7/lib/python3.12/site-packages (from visidata) (8.5.0)
Requirement already satisfied: zipp>=3.20 in ./.pyenv/versions/3.12.7/lib/python3.12/site-packages (from importlib-metadata>=3.6->visidata) (3.20.2)
Requirement already satisfied: six>=1.5 in ./.pyenv/versions/3.12.7/lib/python3.12/site-packages (from python-dateutil->visidata) (1.16.0)
Using cached visidata-3.1-py3-none-any.whl (529 kB)
Installing collected packages: visidata
Successfully installed visidata-3.1
❯ which visidata
/Users/reagle/.pyenv/shims/visidata
 ~
❯ visidata
Traceback (most recent call last):
  File "/Users/reagle/.pyenv/versions/3.12.7/bin/visidata", line 5, in <module>
    from visidata.main import vd_cli
  File "/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/site-packages/visidata/__init__.py", line 157, in <module>
    importFeatures()
  File "/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/site-packages/visidata/__init__.py", line 139, in importFeatures
    vd.importSubmodules('visidata.features')
  File "/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/site-packages/visidata/settings.py", line 508, in importSubmodules
    vd.importModule(pkgname + '.' + module.name)
  File "/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/site-packages/visidata/settings.py", line 491, in importModule
    r = importlib.import_module(pkgname)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/site-packages/visidata/features/errors_guide.py", line 1, in <module>
    from visidata import GuideSheet, vd
ImportError: cannot import name 'GuideSheet' from partially initialized module 'visidata' (most likely due to a circular import) (/Users/reagle/.pyenv/versions/3.12.7/lib/python3.12/site-packages/visidata/__init__.py)
 ~
```

**Comment by reagle** [comment]
Closing re #2561 .

**Comment by anjakefala** [comment]
This should now be fixed @reagle with v3.1.1!
